docs/Analyzer Configuration.md

```diff
@@ -38,7 +38,52 @@ This section documents the list of supported .editorconfig key-value options for
+Configurable Rules: 
+[CA1000](https://docs.microsoft.com/visualstudio/code-quality/ca1000-do-not-declare-static-members-on-generic-types),
```

Comment:
This still renders as a list for classical markdowns but allow for an easier merge (line based).